internal/characters/raiden/charge.go

```diff
@@ -76,6 +76,9 @@ func (c *char) swordCharge(p map[string]int) action.ActionInfo {
 		ax := ai
 		ax.Mult = mult[c.TalentLvlBurst()]
 		ax.Mult += resolveBonus[c.TalentLvlBurst()] * c.stacksConsumed
+		if i < len(chargeSword) - 1 { // Only the last hit has hitlag
```

Comment:
DefHalt also needs to be set to false here

Also should just do `i == 0` here instead of doing an extra len check